cardano-cli/test/cardano-cli-golden/files/golden/help.cli

```diff
@@ -544,7 +544,6 @@ Usage: cardano-cli legacy genesis
                                     | create-cardano
                                     | create
                                     | create-staked
-                                    | hash
```

Comment:
Don't break this. Let's follow our deprecation plan. Print a deprecation warning and after two months we can remove it. It would be nice if CI could warn us but that's a nice to have and not required in this PR.

Reply:
Addressed here: https://github.com/IntersectMBO/cardano-cli/pull/982/commits/a87fbf4ee1aae47967d404ac6b546db8f12cb7ec